src/main/java/com/google/cloud/resourcemanager/OrgPolicyInfo.java

```diff
@@ -0,0 +1,397 @@
+
+import com.google.api.services.cloudresourcemanager.model.OrgPolicy;
+import com.google.api.services.cloudresourcemanager.model.RestoreDefault;
+import com.google.common.base.Function;
+import com.google.common.base.MoreObjects;
+import java.io.Serializable;
+import java.util.List;
+import java.util.Objects;
+
+/** A Google Cloud Resource Manager organization policy metadata object. */
+public class OrgPolicyInfo implements Serializable {
+  static final Function<OrgPolicy, OrgPolicyInfo> FROM_PB_FUNCTION =
+      new Function<com.google.api.services.cloudresourcemanager.model.OrgPolicy, OrgPolicyInfo>() {
+        @Override
+        public OrgPolicyInfo apply(
+            com.google.api.services.cloudresourcemanager.model.OrgPolicy pb) {
+          return OrgPolicyInfo.fromPb(pb);
+        }
+      };
+  static final Function<OrgPolicyInfo, com.google.api.services.cloudresourcemanager.model.OrgPolicy>
+      TO_PB_FUNCTION =
+          new Function<
+              OrgPolicyInfo, com.google.api.services.cloudresourcemanager.model.OrgPolicy>() {
+            @Override
+            public com.google.api.services.cloudresourcemanager.model.OrgPolicy apply(
+                OrgPolicyInfo orgPolicyInfo) {
+              return orgPolicyInfo.toPb();
+            }
+          };
+  private static final long serialVersionUID = 9148970963697734236L;
+
+  private BooleanPolicy booleanPolicy;
+  private String constraint;
+  private String etag;
+  private ListPolicy listPolicy;
+  private RestoreDefault restoreDefault;
+  private String updateTime;
+  private Integer version;
+
+  /** Used in policy_type to specify how booleanPolicy will behave at this resource. */
+  public static class BooleanPolicy implements Serializable {
+    private static final long serialVersionUID = -2133042982786959351L;
+    private final Boolean enforce;
+
+    BooleanPolicy(Boolean enforce) {
+      this.enforce = enforce;
+    }
+
+    public boolean getEnforce() {
+      return enforce;
+    }
+
+    @Override
+    public String toString() {
+      return MoreObjects.toStringHelper(this).add("enforce", getEnforce()).toString();
+    }
+
+    @Override
+    public int hashCode() {
+      return Objects.hash(getEnforce());
+    }
+
+    @Override
+    public final boolean equals(Object obj) {
+      return obj == this
+          || obj != null
+              && obj.getClass().equals(BooleanPolicy.class)
+              && Objects.equals(toPb(), ((BooleanPolicy) obj).toPb());
+    }
+
+    com.google.api.services.cloudresourcemanager.model.BooleanPolicy toPb() {
+      return new com.google.api.services.cloudresourcemanager.model.BooleanPolicy()
+          .setEnforced(enforce);
+    }
+
+    static BooleanPolicy fromPb(
+        com.google.api.services.cloudresourcemanager.model.BooleanPolicy booleanPolicy) {
+      return new BooleanPolicy(booleanPolicy.getEnforced());
+    }
+  }
+
+  /**
+   * Used in policy_type to specify how listPolicy behaves at this resource.
```

Comment:
what does it mean to "behave at this resource"?

Reply:
done